### ISSUE TITLE
[Feature] : Netflix Titles Dataset Visualization

### ISSUE THREAD

**Issue by AKACHI-4** [OpenCodeEra/ML-X#4]
## What ?

I want to create a data visualization notebook for the Netflix Titles dataset to uncover insights like genre distribution, regional availability, and content growth trends over time.

- ### Dataset

add the dataset you find best fit for this issue.

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

<br>
<br>

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by Chandak-Keshav** [comment]
Kindly assign it to me under HacktoberFest'23 @AKACHI-4 
I will surely complete it ASAP

**Comment by AKACHI-4** [comment]
Hey @Chandak-Keshav, we'd be happy to assign this issue to you, but first, please provide a dataset for your work.

**Comment by Chandak-Keshav** [comment]
https://www.kaggle.com/datasets/victorsoeiro/netflix-tv-shows-and-movies/data?select=titles.csv
I was thinking of going with the above dataset available on kaggle. Should I proceed with this dataset @AKACHI-4 ?

**Comment by AKACHI-4** [comment]
> https://www.kaggle.com/datasets/victorsoeiro/netflix-tv-shows-and-movies/data?select=titles.csv I was thinking of going with the above dataset available on kaggle. Should I proceed with this dataset @AKACHI-4 ?

Yeah, sure, @Chandak-Keshav... also, don't forget to take notes while working on the notebook. You can refer to the [Handwritten Digit Recognition model](https://github.com/OpenCodeEra/ML-X/blob/main/Handwritten%20Digit%20Recognition/notebook.ipynb) for the same.

**Comment by Chandak-Keshav** [comment]
I have made a pull request @AKACHI-4 . Kindly review it and let me know if any changes are required. Thanks

**Comment by AKACHI-4** [comment]
@Chandak-Keshav Okay I will...
